scalability_and_performance/low_latency_tuning/cnf-tuning-low-latency-nodes-with-perf-profile.adoc

```diff
@@ -0,0 +1,98 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="cnf-tuning-low-latency-nodes-with-perf-profile"]
+= Tuning nodes for low latency with the performance profile
+include::_attributes/common-attributes.adoc[]
+:context: cnf-low-latency-perf-profile
+
+toc::[]
+
+Tune nodes for low latency by using the cluster performance profile.
+You can restrict CPUs for infra and application containers, configure huge pages and Hyper-Threading, and configure CPU partitions for latency-sensitive processes.
+
+[role="_additional-resources"]
+.Additional resources
+
+* xref:../../scalability_and_performance/low_latency_tuning/cnf-provisioning-low-latency-workloads.adoc#cnf-provisioning-low-latency-workloads[Provisioning real-time and low latency workloads]
+
+[id="cnf-create-performance-profiles"]
+== Creating a performance profile
+
+Learn about the Performance Profile Creator (PPC) and how you can use it to create a performance profile.
+
+[NOTE]
+====
+Currently, disabling CPU load balancing is not supported by cgroup v2. As a result, you might not get the desired behavior from performance profiles if you have cgroup v2 enabled. Enabling cgroup v2 is not recommended if you are using performance profiles.
+====
+
+include::modules/cnf-about-the-profile-creator-tool.adoc[leveloffset=+2]
+
+include::modules/cnf-gathering-data-about-cluster-using-must-gather.adoc[leveloffset=+3]
+
+include::modules/cnf-running-the-performance-creator-profile.adoc[leveloffset=+3]
+
+[role="_additional-resources"]
+.Additional resources
+
+* For more information about the `must-gather` tool,
+see xref:../../support/gathering-cluster-data.adoc#nodes-nodes-managing[Gathering data about your cluster].
+
+include::modules/cnf-how-run-podman-to-create-profile.adoc[leveloffset=+4]
+
+include::modules/cnf-running-the-performance-creator-profile-offline.adoc[leveloffset=+3]
+
+include::modules/cnf-performance-profile-creator-arguments.adoc[leveloffset=+3]
+
+[id="cnf-create-performance-profiles-reference"]
+=== Reference performance profiles
+
+Use the following reference performance profiles as the basis to develop your own custom profiles.
+
+include::modules/installation-openstack-ovs-dpdk-performance-profile.adoc[leveloffset=+3]
+
+include::modules/cnf-telco-ran-reference-design-performance-profile-template.adoc[leveloffset=+3]
+
+include::modules/cnf-telco-core-reference-design-performance-profile-template.adoc[leveloffset=+3]
+
+include::modules/cnf-use-device-interrupt-processing-for-isolated-cpus.adoc[leveloffset=+1]
+
+include::modules/cnf-configuring-workload-hints.adoc[leveloffset=+1]
+
+include::modules/cnf-configuring-power-saving-for-nodes.adoc[leveloffset=+1]
+
+[role="_additional-resources"]
+.Additional resources
+
+* xref:../../scalability_and_performance/low_latency_tuning/cnf-provisioning-low-latency-workloads.adoc#cnf-configuring-high-priority-workload-pods_cnf-provisioning-low-latency[Configuring high priority workload pods]
```

Comment:
This points to a section called "Disabling power saving mode for high priority pods"? Link title should match target.